app/components/package-select-list.js

```diff
@@ -12,8 +12,11 @@ export default SelectList.extend( {
   didRender() {
     this._super(...arguments);
     if (!this.get('didUpdatedOnce') && this.get('pkg') !== null && this.get('pkg.item.packageType') !== null){
-        var packageType = this.get('pkg.packageType') || this.get('pkg.item.packageType');
-        this.set('pkg.notes', packageType.get('name'));
+        var packageType = this.get('pkg.item.packageType');
+
+        if (this.get('pkg.notes') === packageType.get('name') || this.get('pkg.notes') === "" || (this.get('pkg.packageType.name') !== packageType.get('name'))) {
```

Comment:
@abulsayyad123  Would it be better to cache **pkg** and extract out the conditions in a function returning only true or false??